modules/osd-monitoring-troubleshooting-issues.adoc

```diff
@@ -0,0 +1,109 @@
+
+// Module included in the following assemblies:
+//
+// * monitoring/osd-troubleshooting-monitoring-issues.adoc
+
+[id="troubleshooting-monitoring-issues_{context}"]
+= Determining why user-defined project metrics are unavailable
+
+If metrics are not displaying when monitoring user-defined projects, follow these steps to troubleshoot the issue.
+
+.Procedure
+
+. Query the metric name and verify that the project is correct:
+.. From the *Developer* perspective in the OpenShift Container Platform web console, select *Monitoring* > *Metrics*.
```

Comment:
Developer perspective no *Metrics* under *Monitoring*, deleting *Metrics* is ok

Reply:
Deleted it.

Reply:
no, we should not delete it, "Metrics" is a tab here, see the image
https://pasteboard.co/JTkx8lO.png

Reply:
I have changed it back to keep *Metrics* Thank you!